File: Desktop/Models/ApiResponse.cs
using System.Text.Json.Serialization;

namespace FrutNatura.Desktop.Models
{
    public class ApiResponse<T>
    {
        [JsonPropertyName("items")]
        public T Items { get; set; } = default!;

        [JsonPropertyName("totalCount")]
        public int TotalCount { get; set; }

        [JsonPropertyName("page")]
        public int Page { get; set; }

        [JsonPropertyName("pageSize")]
        public int PageSize { get; set; }
    }
}

File: Interface/frutNatura_funci2/app/src/main/java/com/frutnatura/app/api/ApiResponse.java
package com.frutnatura.app.api;

import com.google.gson.annotations.SerializedName;

public class ApiResponse<T> {
    @SerializedName("items")
    public T items;

    @SerializedName("totalCount")
    public int totalCount;

    @SerializedName("page")
    public int page;

    @SerializedName("pageSize")
    public int pageSize;
}
